source/OctoVersion.Core/Configuration/AppSettings.cs

```diff
@@ -37,6 +38,8 @@ public class AppSettings : IAppSettings, IValidatableObject
 
         public string OutputJsonFile { get; set; }
 
+        public string VersionTemplate { get; set; } = "{major}.{minor}.{patch}-{preReleaseTag}.{build}";
```

Comment:
This shouldn't know anything about our internal use cases - it should be structured according to normal SemVer.

```suggestion
        public string VersionTemplate { get; set; } = "{major}.{minor}.{patch}-{preReleaseTag}.{metadata}";
```